.travis.yml

```diff
@@ -16,7 +16,8 @@ before_install:
   - ./fastlane/before_install.sh
 
 script:
-   - fastlane snapshot reset_simulators --force  # Workaround for Travis bug https://github.com/travis-ci/travis-ci/issues/7031
+   - export SNAPSHOT_FORCE_DELETE=true
+   - fastlane snapshot reset_simulators  # Workaround for Travis bug https://github.com/travis-ci/travis-ci/issues/7031
```

Comment:
@rodalfus I'm pretty sure the problem with the build is this line: https://github.com/TCA-Team/iOS/blob/master/.travis.yml#L15

Reply:
Oops nevermind. Looked at the latest log. That wasn't the problem. Weird...

Reply:
@mathiasquintero Actually, you might be right. Looking at the running build, this error message is created:
```
Fetching: public_suffix-3.0.0.gem (100%)
ERROR:  Error installing fastlane:
	public_suffix requires Ruby version >= 2.1.
```
Perhaps that is the reason that a version of fastlane is used that does not support the `--force` option on `reset_simulators` since it's too old.

Reply:
Yup, that seems to be it.
```
[!] The Fastfile requires a fastlane version of >= 2.19.1. You are on 1.111.0.
```

Reply:
Perhaps the node using an outdated image. Maybe try using Xcode8.3

Reply:
@mammuth any idea why the tests might be failing. They are complaining about us using a deprecated API but I'm not sure that's it